### ISSUE TITLE
Backup de preventivo de arquivos

### ISSUE THREAD

**Issue by wagner-sousa** [s3cm44s/osintt001#14]
Alguém consegue prints e vídeos das publicações de todas as redes sociais e com data de publicação antes de haver a exclusão?
1. Não precisa ser a mesma pessoa para todos
2. Não precisa colocar tudo aqui, basta liberar um link para uma pasta pública
3. Basta informar a rede social, perfil e período
4. Motivo: o MP está envolvido e para futuras novas montagens

**Comment by unacachitomas** [comment]
Estou subindo aqui: https://drive.google.com/drive/folders/1Xu2xtmoIqnp_JxpB3nuLMeLT4WeKNAXK?usp=sharing
Quem quiser colocaborar pode solicitar a permissão para editar que libero.
Para que fique em orden cronológica estou utilizando formato mês.dia

**Comment by wagner-sousa** [comment]
@unacachitomas 
Obrigado pelo apoio.
É muito complexo conseguir os vídeos do instagram?

**Comment by unacachitomas** [comment]
não, amanhã vou focar neles e subir tudo. Vou analisar se há manipulação nos vídeos.

**Comment by wagner-sousa** [comment]
@unacachitomas Muito obrigado pelo apoio.

**Comment by Mike-Null** [comment]
Vídeo postado hoje no tiktok @dudapoleza

 

https://user-images.githubusercontent.com/114615977/193718421-1fc61db2-63db-4901-8d6e-80ca1656e2f7.mp4

**Comment by unacachitomas** [comment]
@Mike-Null obrigada, vou fazer o backup dos vídeos no TikTok dela relacionados ao AN também.
